hack/go-vet.sh

```diff
@@ -25,9 +25,11 @@ operator-sdk generate crds --crd-version=v1beta1
 echo "Generating YAML installer"
 # Generate kogito-operator.yaml
 rm kogito-operator.yaml
-declare deploy_files=("./deploy/role.yaml" "./deploy/service_account.yaml" "./deploy/role_binding.yaml" "./deploy/operator.yaml")
+declare rbac_files=("./deploy/clusterrole.yaml" "./deploy/service_account.yaml" "./deploy/clusterrole_binding.yaml")
+cat ./deploy/namespace.yaml > kogito-operator.yaml; printf "\n---\n" >> kogito-operator.yaml
 for yaml in deploy/crds/*_crd.yaml; do cat "${yaml}" >> kogito-operator.yaml; printf "\n---\n" >> kogito-operator.yaml; done
-for yaml in "${deploy_files[@]}"; do cat "${yaml}" >> kogito-operator.yaml; printf "\n---\n" >> kogito-operator.yaml; done
+for yaml in "${rbac_files[@]}"; do cat "${yaml}" >> kogito-operator.yaml; printf "\n---\n" >> kogito-operator.yaml; done
+sed '4 a \ \ namespace: operators' ./deploy/operator.yaml >> kogito-operator.yaml
```

Comment:
@Kaitou786 This should be fixed as well.

Reply:
Ah yes, thanks. Changed :)